grails-app/controllers/org/pih/warehouse/core/StockMovementItemsParamsCommand.groovy

```diff
@@ -11,6 +11,13 @@ class StockMovementItemsParamsCommand extends PaginationCommand implements Valid
 
     Boolean refreshPicklistItems = true
 
+    @Override
+    Integer getMax() {
+        // Overriding getter for max value from pagination command to be able
+        // to get all of the items, instead of just 10 (10 was a default value in pagination command)
+        return super.originalMax
```

Comment:
it looks like 10 is the default value for the original getMax if "max" is undefined. Is the issue that the client is not providing a max in the request? I think having the client provide a value would be better than bypassing our global max value for pagination.

Or is the issue that the client is trying to provide a value >100, which isn't allowed because of the hard cap? If so, it seems this solution is just a roundabout way to bypass doing any pagination on the request. If that's the case, I'd suggest one of the following:

1) Don't make this a PaginationCommand
2) Have the client properly support pagination of the items, and while it's fetching the stock movement, loop until there's no more items to fetch.

2 is my preference if possible since it's generally good to keep our requests to a known size.